ecs-init/docker/docker_test.go

```diff
@@ -29,8 +29,8 @@ import (
 // Note: Change this value every time when a new bind mount is added to
 // agent for the tests to pass
 const (
-	expectedAgentBindsUnspecifiedPlatform = 17
-	expectedAgentBindsSuseUbuntuPlatform  = 15
+	expectedAgentBindsUnspecifiedPlatform = 18
```

Comment:
The test also needs to be updated to check the new added bind. The expectKey part.

Reply:
will do